src/components/entry/EntryDetails.js

```diff
@@ -151,6 +174,23 @@ const EntryDetails = () => {
         {typesHeader}
         <Description>
           <p>{locationData.description}</p>
+          <IconBesideText bold onClick={handleAddressClick} tabIndex={0}>
+            <Map color={theme.secondaryText} size={20} />
+            <LocationText>{address}</LocationText>
+          </IconBesideText>
+          {locationData.no_season ||
+            (locationData.season_start && locationData.season_stop && (
```

Comment:
nesting's a bit off here. the first ending parentheses should come after `season_stop`

Reply:
Actually just make a variable in the body of entry details. `const hasSeasonality = locationData.no_season || (locationData.season_start && locationData.season_stop)`.

Reply:
`hasSeasonality && JSX`

Reply:
good catch, Quarantine!

Reply:
oh damn these additional comments just loaded. glad i saw